project/JdkOptions.scala

```diff
@@ -27,6 +27,19 @@ object JdkOptions extends AutoPlugin {
 
   lazy val specificationVersion: String = sys.props("java.specification.version")
 
+  object JavaVersion {
+    val majorVersion: Int = {
```

Comment:
I am fine with that, but it will be great if we have a better solution for this.